psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="false"
```

Comment:
Don't commit it, but what happens with `totallyTyped="true"`?

Reply:
Ha, ok, not gonna be able to fix that soon then :+1: 

Thanks for trying it out!

Reply:
you are very welcome : )